uber/models.py

```diff
@@ -1674,9 +1674,13 @@ def rcpt_name(self):
             return self.fk.full_name
 
     @property
+    def is_html(self):
+        return '<body' in self.body
```

Comment:
Some emails have an basic ``<body>`` tag and others have attributes, e.g. ``<body style="margin:0px">``, which is why we check for ``<body`` and not ``<body>``.